README.md

```diff
@@ -7,7 +7,8 @@ This process wil have four steps:
   1.b. commit this file
   1.c. accept changes and merge with Master
   
-2:  
+2:  After a day of being logged off, go back and try some more updates
```

Comment:
Does it just have to be a day?  Or would simply logging off and on again suffice?